tests/dictionary/test_canopen_dictionary.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from tests.conftest import read_config
+from ingenialink.canopen.dictionary import CanopenDictionary
+
+
+@pytest.mark.smoke
+def test_get_register_dictionary(read_config):
+    file_test_path = read_config['canopen']['dictionary']
```

Comment:
That should be a test dictionary, if you make it configurable could fail, for example, if the number of registers is different.